src/lib/pull-request.coffee

```diff
@@ -25,6 +25,15 @@ pr = (token, branch, body, cb) ->
   If I do something wrong, [blame a human](https://github.com/cfpb/hubot-capital-framework/issues/new).
 
   ![kitten gif](http://thecatapi.com/api/images/get?format=src&type=gif)
+
+  ## You are not
+
+  After this PR is merged, you should be prepared to update the CF dependencies I've released in the following projects:
+
+  - [ ] [capital-framework gh-pages branch](https://github.com/cfpb/capital-framework/tree/gh-pages)
```

Comment:
It might be worth explicitly explaining that this is what creates CF's docs site.

Reply:
If you're running the command in chat I sure hope you know that already.

Reply:
One would hope, but y'know… never hurts to be explicit :)